Tests/AppTests/AnalyzeErrorTests.swift

```diff
@@ -21,20 +21,23 @@ import Testing
 import Vapor
 
 
+extension AllTests {
+    @Suite struct AnalyzeErrorTests {
+        let socialPosts = LockIsolated<[String]>([])
+    }
+}
```

Comment:
Some of the nested `@Suite`s can't be defined in `AllTests.swift`, because they hold properties (which can't be declared in extensions).